### PR TITLE
theme: add papercolor light

### PR DESCRIPTION
**Pull request by uncomfyhalomacro**
based on https://github.com/NLKNguyen/papercolor-theme

### CODE REVIEW

**Comment by n0s4** [comment]
Do you want to add some of the markup scopes? There is no highlighting for markdown files.

**Comment by uncomfyhalomacro** [comment]
i will. me will ping once PR is ready. sorry for the incompleteness of the PR. 😅

**Comment by uncomfyhalomacro** [comment]
> Do you want to add some of the markup scopes? There is no highlighting for markdown files.

Added :smile: thankies! PR ready for review

**Review by pickfire — APPROVED** [review]
Thanks.

File: runtime/themes/papercolor-light.toml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+"comment" = { fg = "bright0", modifiers = ["dim"] }
+"module" = "#af0000"
+"constant" = "#5f8700"
+"constant.builtin" = "#5f8700"
+"constant.numeric" = "#d75f00"
+"constant.character.escape" = { fg = "#8700af", modifiers = ["bold"]}
+"operator" = { fg = "regular4", modifiers = ["bold"]}
+
+"label" = { fg = "selection_background", modifiers = ["bold", "italic"] }
+
```
Comment (pickfire) [inline runtime/themes/papercolor-light.toml:51]: I believe it is also missing `diff` colors, but either way I will merge for now.

**Comment by the-mikedavis** [comment]
@uncomfyhalomacro could you add a screenshot to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

**Pull request merged**